src/main/java/com/jcraft/jsch/OpenSSHConfig.java

```diff
@@ -74,6 +77,9 @@
  */
 public class OpenSSHConfig implements ConfigRepository {
 
+  private static final List<String> keysWithListAdoption = Stream.of("KexAlgorithms", "Ciphers", "HostbasedAcceptedAlgorithms", 
+  "HostKeyAlgorithms", "MACs", "PubkeyAcceptedAlgorithms", "CASignatureAlgorithms").map(String::toUpperCase).collect(Collectors.toList());
```

Comment:
Do we need to handle the legacy names like `PubkeyAcceptedKeyTypes` & `HostbasedKeyTypes` that were used until OpenSSH 8.4?

Reply:
And `HostbasedAcceptedKeyTypes` as well?

Reply:
I will add PubkeyAcceptedKeyTypes, but HostbasedKeyTypes is not handled anywhere in Jsch